test/integration/ovs/ofctrl_test.go

```diff
@@ -355,6 +355,52 @@ func TestBundleErrorWhenOVSRestart(t *testing.T) {
 	require.Equal(t, 0, failCount, "No fail case expected")
 }
 
+// TestReconnectOFSwitch verifies that the OpenFlow connection to OVS can be restored, even when OVS is down for a long
+// amount of time.
+func TestReconnectOFSwitch(t *testing.T) {
+	br := "br07"
+	err := PrepareOVSBridge(br)
+	require.Nil(t, err, fmt.Sprintf("Failed to prepare OVS bridge: %v", err))
+	defer func() {
+		err = DeleteOVSBridge(br)
+	}()
+
+	bridge := binding.NewOFBridge(br)
+	reconnectCh := make(chan struct{})
+	var connectCount int
+	go func() {
+		for {
+			select {
+			case <-reconnectCh:
+				connectCount++
+			}
+		}
```

Comment:
Same as below, no need to use select when there's only one condition, it should be simplified as:
```
go func() {
        for range reconnectCh {
                connectCount++
        }
}
```

Reply:
Got it, updated.